deb/common/rules

```diff
@@ -85,7 +85,6 @@ override_dh_auto_install:
 
 	# docker-ce-rootless-extras install
 	install -D -m 0755 /usr/local/bin/rootlesskit debian/docker-ce-rootless-extras/usr/bin/rootlesskit
-	install -D -m 0755 /usr/local/bin/rootlesskit-docker-proxy debian/docker-ce-rootless-extras/usr/bin/rootlesskit-docker-proxy
```

Comment:
Should still be installed for v27 and older

Reply:
Oh, the packaging isn't branched per release? I'll take a look tomorrow, thank you.

Reply:
27.x branch was created; https://github.com/docker/docker-ce-packaging/tree/27.x